slurm_ops_manager/slurm_ops_base.py

```diff
@@ -428,6 +428,17 @@ def render_nhc_config(self, extra_configs=None) -> bool:
             logger.error(f"#### Error rendering NHC.conf: {e}")
             return False
 
+    def render_nhc_wrapper(self, params: str):
+        """Render the /usr/sbin/omni-nhc-wrapper script."""
+
+        target = Path("/usr/sbin/omni-nhc-wrapper")
+        context = {"nhc_params": params}
+        environment = Environment(loader=FileSystemLoader(TEMPLATE_DIR))
+        template = environment.get_template('omni-nhc-wrapper.tmpl')
+
+        target.write_text(template.render(context))
+        target.chmod(0o755) # everybody can read/execute, owner can write
```

Comment:
why are there permissions required?

Reply:
The owner needs to be able to read/write/execute. Other people should no be able to edit, but they should be able to view and execute.